config-overrides.js

```diff
@@ -0,0 +1,32 @@
+const webpack = require("webpack");
+const ModuleScopePlugin = require("react-dev-utils/ModuleScopePlugin");
+
+module.exports = function override(config) {
+  config.resolve.plugins = config.resolve.plugins.filter((plugin) => !(plugin instanceof ModuleScopePlugin));
+  // https://github.com/lingui/js-lingui/issues/1195
+  config.module.rules[0].oneOf.splice(2, 0, {
```

Comment:
Perhaps we should add a more detailed comment on what rule is changed here and why?

Or more explicitly, redefine this like:
config.module.rules[0] = {...} ?

Reply:
https://github.com/gmx-io/gmx-interface/pull/244/commits/33b99b68f2e270434cdbc3ffbe6d854487393ab5

Refactored and added comment explaining the loader we are adding to webpack ocnfig.